packages/kbn-test/src/functional_tests/cli/run_tests_cli.js

```diff
@@ -0,0 +1,63 @@
+import getopts from 'getopts';
+import { runTests } from '../../';
+import {
+  FUNCTIONAL_CONFIG_PATH,
+  API_CONFIG_PATH,
+} from '../../functional_tests/lib';
+
+const defaultConfigPaths = [FUNCTIONAL_CONFIG_PATH, API_CONFIG_PATH];
+
+/**
+ * Run servers and tests for each config
+ * Only cares about --config option. Other options
+ * are passed directly to functional_test_runner, such as
+ * --bail, --verbose, etc.
+ * In the future, this method will accept --tests-only,
+ * --servers-only.
+ * @param {configPaths}     array of paths to configs
+ */
+export function runTestsCli() {
+  const {
+    configs,
+    help,
+  } = processArgv();
+
+  if (help) return displayHelp();
+
+  runTests(configs);
+}
+
+
+function processArgv() {
+  const options = getopts(process.argv.slice(2)) || {};
+  const configs = [];
+
+  try {
+    let configOptions = options.config;
+    configOptions = typeof configOptions === 'string'
+      ? [configOptions]
+      : configOptions;
+    configs.push(...configOptions);
+  } catch (err) {
+    configs.push(...defaultConfigPaths);
+  }
+
+  return {
+    configs,
+    help: options.help,
+    rest: options._,
+  };
+}
+
+
+function displayHelp() {
+  console.log();
```

Comment:
__nit:__ how about using `dedent` and single `console.log` instead like is done here https://github.com/elastic/kibana/blob/master/packages/kbn-pm/src/cli.ts#L14?

Reply:
Done